Sources/PortalSwift/Utils/Errors/PortalMpcError.swift

```diff
@@ -8,23 +8,26 @@
 import Foundation
 
 public class PortalMpcError: LocalizedError, CustomStringConvertible, Equatable {
-  public var code: Int
-  public var message: String
+  @available(*, deprecated, message: "Use `id` instead.")
+  public var code: Int?
+  public var id: String?
+  public var message: String?
 
   init(_ error: PortalError) {
     self.code = error.code
+    self.id = error.id
     self.message = error.message
   }
 
   public var errorDescription: String {
-    return "PortalMpcError -code: \(self.code) -message: \(self.message)"
+    return "PortalMpcError -id: \(self.id ?? "") -message: \(self.message ?? "")"
```

Comment:
we are still checking for code but here we use id for the message. should we continue to use code for the message until we cut over?

Reply:
add a default value of unknown here instead of empty string